CONTRIBUTING.md

```diff
@@ -1,15 +1,25 @@
 [Back to README](README.md)
 
-## Contributing
+## Contributing to CircuitVerse
+
+- Thanks for checking out our guide. We'll be glad to hear from you.
+- Interested in making a contribution? Read on!
+
+## How to Contribute
 
 When you are ready to start work on a issue:
 
 - Let us know by leaving a comment on the issue. (Also let us know later if you are no longer working on it.)
 
 - Once you are assigned the issue (or you have claimed the issue) only then proceed to making the Pull Request. This will help avoid multiple PRs pertaining to the same issue.
 
+If you don't see your idea listed, do one of the following:
+
+* **If your contribution is minor,** such as a typo fix, open a pull request.
+* **If your contribution is major,** such as a new guide, start by opening an issue first. That way, other people can be also involved in the discussion before you do any work.
```

Comment:
*such as new feature/enhancement* maybe can be used, `guide` sounds a little off.

Reply:
will do so